src/algorithms/ctmrg.jl

```diff
@@ -96,11 +91,21 @@ function MPSKit.leading_boundary(envinit, state, alg::CTMRG)
     )
     env′, = ctmrg_iter(state, env, alg_fixed)
     envfix = gauge_fix(env, env′)
-    check_elementwise_convergence(env, envfix; atol=alg.tol^(3 / 4)) ||
+    check_elementwise_convergence(env, envfix; atol=alg.tol^(1 / 2)) ||
         @warn "CTMRG did not converge elementwise."
     return envfix
 end
 
+#very ugly fix to avoid Zygoting @plansor during fermionic gauge fix
+@generated function MPSKit.transfer_right(v::AbstractTensorMap{S,1,N₁}, A::GenericMPSTensor{S,N₂},
+    Ā::GenericMPSTensor{S,N₂}) where {S,N₁,N₂}
+        t_out = MPSKit.tensorexpr(:v, -1, -(2:(N₁ + 1)))
+        t_top = MPSKit.tensorexpr(:A, (-1, reverse(3:(N₂ + 1))...), 1)
+        t_bot = MPSKit.tensorexpr(:Ā, (-(N₁ + 1), reverse(3:(N₂ + 1))...), 2)
+        t_in = MPSKit.tensorexpr(:v, 1, (-(2:N₁)..., 2))
+        return :(return @tensor $t_out := $t_top * conj($t_bot) * $t_in)
+end
```

Comment:
This could be made less ugly by explicitly checking that the braiding is symmetric, otherwise this definition also breaks anyonic mps code. Alternatively, (and way nicer) is to implement the reverse rule for this operation (which should be `transfer_left`)

Reply:
In general this is also not correct for fermions and one would need P's in there when doing something MPS optimization or whatever oriented and using @tensor. However, here you don't really need them as you can absorb everything in fixed-points and random bottom layer MPS. 

How would you go about the transfer_left rrule option?

Reply:
No, I mean this implementation breaks the non-ad pass of MPSKit. If you now have both MPSKit and PEPSKit loaded, you will no longer be able to do Fibonacci anyons.

Also, I think I was too quick with thinking it is just the other transfer, as you also need the derivatives to the transfer matrix

Reply:
yes I know, but also even for fermions the forward pass would be broken now. However this broken fermionic is still ok here due to the specific nature of what we're trying to do here (find these phases). In short: I fully agree that it is a very messy fix :p

Reply:
Ok, in any case, I changed the implementation to avoid using MPSKit, as I don't think fixing one thing while breaking another is really a fix :p